bin/configure-app-service.sh

```diff
@@ -13,6 +13,8 @@ set -euo pipefail
 APP_NAME=$1
 ENVIRONMENT=$2
 
+DNS_ZONE=platform.navateam.com
+
```

Comment:
What is "DNS zone"? Is it the same as domain name?

Could we pull this from project-config/main.tf and then do

```
DNS_ZONE=$(terraform -chdir infra/project-config out -raw dns_zone)
```